### PR TITLE
Create app functions list

### LINKED ISSUE #31: Create userflow

### PR DESCRIPTION
**Pull request by naila135**
Resolve #31

### CODE REVIEW

File: Specification/UserFlow/AppFunctions.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+This is a list of features that the application will receive in the MVP version.<br>
```
Comment (sahlet-official) [inline Specification/UserFlow/AppFunctions.md:1]: Change "receive" -> "give the user"

File: Specification/UserFlow/AppFunctions.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+The functions list and usage scenarios was sourced from [UserStoryWithoutApp](./UserStoryWithoutApp.md) and an interview with the Product Owner.<br>
+A set of functions for MVP:
+- the application tracks the moment of falling asleep;
+- the application tracks the amount of sleep;
+- the application records the nature of sleep without trackers;
```
Comment (sahlet-official) [inline Specification/UserFlow/AppFunctions.md:6]: change line to
the application superficially records sleep variability without external tracker devices;

File: Specification/UserFlow/AppFunctions.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- the application records the moment of awakening;
+- the user can refine and / or change additional data applications;
```
Comment (sahlet-official) [inline Specification/UserFlow/AppFunctions.md:8]: Change please to
the user can edit and supplement sleep data records;

File: Specification/UserFlow/AppFunctions.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+- the user can add notes to each dream and thus keep a journal;
```
Comment (sahlet-official) [inline Specification/UserFlow/AppFunctions.md:9]: Change please to
the user can add comments to each sleep record;

File: Specification/UserFlow/AppFunctions.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+- the application visualizes the received data in the form of graphs;
```
Comment (sahlet-official) [inline Specification/UserFlow/AppFunctions.md:10]: the application visualizes the data series in the form of different types of graphs;

File: Specification/UserFlow/AppFunctions.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+- analysis data can be viewed in the form of a log;
```
Comment (sahlet-official) [inline Specification/UserFlow/AppFunctions.md:11]: Change please to
stored sleep records can be viewed in the form of a log;

File: Specification/UserFlow/AppFunctions.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+<empty line>
+  
+A set of usage scenarios for MVP:
+- start monitoring sleep;
+- end sleep monitoring;
+- let the device know that you are not sleeping;
+- view daily report;
+- complete the sleep log;
```
Comment (sahlet-official) [inline Specification/UserFlow/AppFunctions.md:19]: Change please to
edit/fill and confirm the sleep record;

File: Specification/UserFlow/AppFunctions.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+- edit sleep data;
```
Comment (sahlet-official) [inline Specification/UserFlow/AppFunctions.md:20]: Remove line please

**Review by sahlet-official — CHANGES_REQUESTED** [review]

File: Specification/UserFlow/AppFunctions.md (hunk 1 above)
Comment (naila135) [reply Specification/UserFlow/AppFunctions.md]: done, but changed a little to make it grammaticaly correct

File: Specification/UserFlow/AppFunctions.md (hunk 2 above)
Comment (naila135) [reply Specification/UserFlow/AppFunctions.md]: done

File: Specification/UserFlow/AppFunctions.md (hunk 3 above)
Comment (naila135) [reply Specification/UserFlow/AppFunctions.md:8]: done

File: Specification/UserFlow/AppFunctions.md (hunk 4 above)
Comment (naila135) [reply Specification/UserFlow/AppFunctions.md]: done

File: Specification/UserFlow/AppFunctions.md (hunk 5 above)
Comment (naila135) [reply Specification/UserFlow/AppFunctions.md]: done

File: Specification/UserFlow/AppFunctions.md (hunk 8 above)
Comment (naila135) [reply Specification/UserFlow/AppFunctions.md]: done

**Review by sahlet-official — APPROVED** [review]

**Pull request merged**